network/internal/p2pfixtures/fixtures.go

```diff
@@ -10,14 +10,10 @@ import (
 	"time"
 
 	addrutil "github.com/libp2p/go-addr-util"
-	dht "github.com/libp2p/go-libp2p-kad-dht"
```

Comment:
Moving some test fixtures to `p2p/test/fixtures.go` so that they become accessible from `insecure` package (for BFT testing).